kinesis/README.md

```diff
@@ -0,0 +1,104 @@
+# Streaming database changes to Amazon Kinesis using Debezium
+
+[Debezium](http://debezium.io/) was a set of Apache Kafka connectors that allows streaming of change events from different database. Using [Debezium Embedded](http://debezium.io/docs/embedded/) it is possible to stream database changes to an arbitrary destination and not be limited to Kafka broker only.
+
+This demo shows how to stream changes from MySQL database running on a local machine to aa Amazon [Kinesis](https://aws.amazon.com/kinesis/data-streams/) stream.
+
+## Pre-requisities
+* Java 8 development environment
+* Local Dockar installation to run source database
+* an Amazon [AWS](https://aws.amazon.com/) account
+* [AWS CLI](https://aws.amazon.com/cli/) client
+
+## How to run
+### Start MySQL source database
+We will start a pre-populated MySQL database that is the same as used by Debezium [tutorial](http://debezium.io/docs/tutorial/).
+
+```mvn docker:run```
+
+### Initialize Kinesis stream
+We suppose that you have already executed `aws configure` as descirbed in AWS CLI installation guide.
+
+### Create the Kinesis stream
+
+```aws kinesis create-stream --stream-name debezium --shard-count 1```
+
+We are using only one shard to keep the events from database in ordered sequence. If you are not interested in a total order but the order per table is enough you can modify the code to use the table name as a sharding key and increase the number of shards.
```

Comment:
So Kinesis does guarantee order of messages within a shard?

Reply:
Thinking more about that, what would be needed to have a stream per source table, more closely mirroring the behavior you'd get with Kafka by default?